### PR TITLE
Hrgym panda

### PR DESCRIPTION
**Pull request by Sh0cktr4p**
Renamed robot parameter file: panda_new -> panda

### CODE REVIEW

**Comment by Sh0cktr4p** [comment]
I am not sure though why there are changes in the LongTermPlanner

**Comment by JakobThumm** [comment]
There shouldn't be any changes to the LTP imo.

**Comment by JakobThumm** [comment]
Apparently, the hrgym_panda branch is on an old version of LTP.
Correct is c52e145a2c8a1c30f72a251350540e266824e0c5 or f8865b0a7556b2df84bd6ec1461205c3a8955d74 (no difference)
So, your changes are correct.

File: safety_shield/config/robot_parameters_panda.yaml [hunk 1]
```diff
@@ -5,11 +5,6 @@ nb_joints: 7
 secure_radius: 0.02
 
 transformation_matrices: [
-    1, 0, 0, 0,
```
Comment (JakobThumm) [inline safety_shield/config/robot_parameters_panda.yaml:8]: Why don't we need this?

File: safety_shield/config/robot_parameters_panda.yaml [hunk 2]
```diff
@@ -38,21 +33,26 @@ transformation_matrices: [
...
+    0, 0, 0, 1,
     ### 
     ]
 
 enclosures: [
 
-    -0.041271, -0.000011589, 0.07,
-    -0.041271, -0.000011589, 0.07,
-    0.13543,
+    #-0.041271, -0.000011589, 0.07,
```
Comment (JakobThumm) [inline safety_shield/config/robot_parameters_panda.yaml:53]: Why don't we need this?

**Pull request merged**